website/pages/en/index.js

```diff
@@ -84,54 +74,247 @@ class Index extends React.Component {
+          }
+          return <div>Loading</div>;
+        }}
+    />
+  )
+}
+                  `}
+                  </code>
+                </pre>
+              </div>
+
+              <div>
+                <h2>Fragment Container</h2>
+                <p>
+                  Step two is to render a tree of React components which includes <code>FragmentContainers</code>, <code>PaginationContainers</code> and <code>RefetchContainers</code>.
+                </p>
+                <p>
+                  The most common are <code>FragmentContainers</code> which  does not directly fetch data, but instead declares a *specification* of the data needed for rendering a component. 
+                </p>
+                <p>
+                  Relay will then guarantee that this data is available *before* rendering occurs.
+                </p>
+              </div>
+              <div>
+                <pre>
+                  <code>
+                  
+                  {`
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Link, Image, Name, Bio, View,} from "./views"
+
+class ArtistHeader extends React.Component {
+  render() {
+    const { name, href, image, bio } = this.props.artist
+    const imageUrl = image && image.url
+
+    return (
+      <Link href={href}>
+        <Image imageUrl={imageUrl} />
+        <View>
+          <Name>{name}</Name>
+          <Bio>{bio}</Bio>
+        </View>
+      </Link>
+    )
+  }
+}
+
+export default createFragmentContainer(
+  Artist,
+  graphql\`
+    # When an Artist is requested, explicitly
+    # expose these fields to only the above Component
+    fragment ArtistItem_artist on Artist {
+      href
+      bio
+      name
+      image {
+        url
+      }
+    }
+  \`
+)
+                  `}
+                  </code>
+                </pre>
+              </div>
+            </div>
+          </Container>
+
+          <Container className='textSection graphqlSection' background="light">
+            <h2>GraphQL Best Practices Baked In</h2>
+            <h3>
+              To get the most from Relay’s features, you’ll want your GraphQL server to conform to the Pagination and Global identiifier best practices.
+            </h3>
+            <GridBlock
+              layout="threeColumn"
+              contents={[
                 {
-                  content:
-                    'GraphQL is written next to the views that rely on them. Relay aggregates queries into efficient network requests.',
-                  image: '',
-                  imageAlign: 'top',
-                  title: 'Colocation',
+                  title: 'Fragments',
+                  content: `<p>A GraphQL fragment is a way to name a sub-query inside a part of your GraphQL query.</p><p>Relay uses Fragments to  component to just the data it uses.</p><p>See the <a href=${siteConfig.baseUrl + 'docs/' + this.props.language + "/fragment-container"}>Fragment docs</a></p>`
```

Comment:
Nit - double space between "to" and "component". But it won't show up in the UI.

Reply:
"Relay uses Fragments to  component to just the data it uses." - this sentence seems broken.